play/src/front/Components/Input/InputSwitch.svelte

```diff
@@ -14,7 +14,7 @@
     {#if labelPosition === "top" && label}
         <label for={uniqueId} class="input-label">{label}</label>
     {/if}
-    <label class="inline-flex cursor-pointer items-center relative">
+    <label class="inline-flex cursor-pointer items-center relative mt-3">
```

Comment:
Do we really want a margin top 3 on every input switch? Is it not something we should add somewhere else? (honest question)